src/create-ad-slot.ts

```diff
@@ -148,28 +148,21 @@ const createClasses = (
  */
 const concatSizeMappings = (
 	defaultSizeMappings: SizeMapping,
-	optionSizeMappings: CreateSlotOptions['sizes'],
-): SizeMapping => {
-	if (!optionSizeMappings) return defaultSizeMappings;
-	const concatenatedSizeMappings: SizeMapping = { ...defaultSizeMappings };
-	const optionDevices = Object.keys(optionSizeMappings); // ['mobile', 'desktop']
-
-	for (let i = 0; i < optionDevices.length; i++) {
-		const device = optionDevices[i] as keyof SizeMapping;
-		if (optionSizeMappings[device] && defaultSizeMappings[device]) {
-			const optionSizeMappingsForDevice = optionSizeMappings[device];
-			const defaultSizeMappingsForDevice = defaultSizeMappings[device];
-
-			if (defaultSizeMappingsForDevice && optionSizeMappingsForDevice) {
-				// TODO can we do concatenatedSizeMappings[device]?.concat ?
-				concatenatedSizeMappings[device] = (
-					concatenatedSizeMappings[device] ?? []
-				).concat(optionSizeMappingsForDevice);
+	optionSizeMappings?: SizeMapping,
```

Comment:
would defaulting to `{}` instead of the coalesce bellow be more readable?
```suggestion
	optionSizeMappings: SizeMapping = {},
```

Reply:
Agreed, Much nicer 👍